resources/istio/files/job-script.sh

```diff
@@ -1,8 +1,14 @@
-
+#!/bin/bash -e
 if [ -f "/etc/istio/overrides.yaml" ]; then
   #New way: just merge default IstioControlPlane definition with a user-provided one.
-  printf "istioctl manifest apply -f /etc/istio/config.yaml -f /etc/istio/overrides.yaml\n"
-  istioctl manifest apply -f /etc/istio/config.yaml -f /etc/istio/overrides.yaml
+  yq merge -a -x /etc/istio/config.yaml /etc/istio/overrides.yaml > /etc/combo.yaml
+  kubectl create cm "${CONFIGMAP_NAME}" -n "${NAMESPACE}" \
```

Comment:
Won't this break during the upgrade in the future? `kubectl create` returns error like `configmap kyma-istio-control-plane-config` already exists. If we want to keep it up to date we should do sth like `kubectl create cm ..... --dry-run -o yaml | kubectl apply -f -`